src/views/ProposalList.vue

```diff
@@ -31,11 +31,13 @@
     <b-row>
       <b-col>
         <b-form inline @submit="onSubmit">
-          <label class="mr-sm-2" for="input-proposal-active">Proposal is active:</label>
+          <label class="mr-1" for="input-proposal-active">Proposal is active:</label>
           <b-form-select id="input-proposal-active" v-model="queryParams.active" :options="proposalActiveOptions" />
-          <label class="m-sm-2" for="input-proposal-semester">Semester:</label>
+          <label class="m-1" for="input-proposal-semester">Semester:</label>
           <b-form-select id="input-proposal-semester" v-model="queryParams.semester" :options="semesterOptions"></b-form-select>
-          <b-button type="submit" class="m-sm-1" variant="outline-primary" :disabled="isBusy">Filter</b-button>
+          <label for="select-proposal-tag" class="m-1">Proposal tag:</label>
+          <b-form-select id="select-proposal-tag" v-model="queryParams.tag" :options="tagOptions" />
```

Comment:
this should probably be a multi-select if we change the filter to be a multiple choice filter